### DISCUSSION TITLE
Spike Prime Sensors on EV3?

### DISCUSSION THREAD

**Discussion by shoringit** [pybricks/support#1237] [Q&A] [answered]
There are adapters available to supposedly allow the spike prime sensors to work on the EV3.  However, I haven't found any code where it's actually been done in practice.  I would like to use the force sensor on the EV3, but need some direction if it's possible and if so, how to achieve it...

Thanks,

Danny

**Comment by laurensvalk** [comment]
The newer sensors use various new protocol features, so it's possible it doesn't get detected out of the box.

**Reply by shoringit** [reply]
No, it's not detecting it, but that doesn't scare me off necessarily.  The problem I have is that I've seen no examples, etc. for how it could be used on the EV3 under pybricks.  I think people have created blocks for EV3-G, so that makes me think it's possible...

Looking in the documentation, I see that it's used on the Spike prime devices under pybricks.pupdevices.  I can't import anything from under pupdevices.  I'm guessing it's because I'm on an EV3.  Or is there a way I can access these functions on the EV3?

Danny

**Reply by laurensvalk** [reply, marked as answer]
This could work if we ever upgraded EV3 to a bare-metal firmware like we have on SPIKE, but there are no plans for doing this right now.

**Reply by shoringit** [reply]
That's what I was afraid of - thanks for the input.

Danny
